src/rpc/server.cpp

```diff
@@ -308,7 +308,7 @@ static UniValue getrpcinfo(const JSONRPCRequest& request)
     for (const RPCCommandExecutionInfo& info : g_rpc_server_info.active_commands) {
         UniValue entry(UniValue::VOBJ);
         entry.pushKV("method", info.method);
-        entry.pushKV("duration", GetTimeMicros() - info.start);
```

Comment:
FYI this is considered breaking change, however you could add a new field. Also, `ms` stands for milliseconds, and the unit here is microseconds.